text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,116 @@
+# RFC-0015: Market Design Revisit
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 05.08.2023                                                                                  |
+| **Description** | This RFC refines the previously proposed mechanisms involving the various Coretime markets and presents an integrated framework for harmonious interaction between all markets.                                                       |
+| **Authors**     | Jonas Gehrlein                                                                              |
+
+## Summary
+
+This document is a proposal for restructuring the bulk markets in the Polkadot UC's coretime allocation system to improve efficiency and fairness. The proposal suggests separating the `BULK_PERIOD` into `MARKET_PERIOD` and `RENEWAL_PERIOD`, allowing for a market-driven price discovery through a clearing price Dutch auction during the `MARKET_PERIOD` followed by renewal offers at the `MARKET_PRICE` during the `RENEWAL_PERIOD`. The new system ensures synchronicity between renewal and market prices, fairness among all current tenants, and efficient price discovery, while preserving price caps to provide security for current tenants. It also discusses the initiation of the instantanous market with a discount and the possibility of long-term leases.
+
+## Motivation
+
+While the initial [RFC-1](https://github.com/polkadot-fellows/RFCs/blob/6f29561a4747bbfd95307ce75cd949dfff359e39/text/0001-agile-coretime.md) has provided a robust framework for Coretime allocation within the Polkadot UC, this proposal builds upon its strengths and uses many provided building blocks to address some areas that could be further improved. 
+
+In particular, this proposal introduces the following changes:
+- It introduces a `RESERVE_PRICE` that anchors all markets, promoting price synchronicity among it's substitute goods (flexible/renewed/instantanous coretime). 
+    - This reduces complexity.
+    - This makes sure all tenants pay the same for coretime.
```

Comment:
It's unclear what "all tenants" means here, but coretime is likely to change in value based on volatile factors. While it seems reasonable to give the same price to all _bulk purchasers_ of coretime, the secondary market should set the price otherwise.

Phrased as broadly as it is here, this is a non-goal of the system

Reply:
The clearing price dutch auction resolves such that all bidders in that market pay the same price (`MARKET_PRICE`) at the end. All entities that renew their cores would also pay the same price. Renewal prices differ from the  `MARKET_PRICE` by `RENEWAL_DISCOUNT`. I'll make that more clear.